backend/DataAccessLayer/Enums/QuantityType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DataAccessLayer.Enums
+{
+    public enum QuantityType
```

Comment:
O kam reikia tokio tipo kaip kilogramai ar litrai?

Reply:
I guess it would be good to include them just in case.